docs/modules/ROOT/pages/user-code-deployment.adoc

```diff
@@ -82,6 +82,8 @@ include::ROOT:example$/hazelcast-user-code-deployment-azure.yaml[]
 <1> The bucket URI where backup data will be stored in
 <2> Name of the secret with credentials for accessing the Bucket URI.
 
+NOTE: Install public buckets using the `remoteURLs` field over HTTP.
```

Comment:
Do we really need this note? For me, it seems a bit redundant. I mean, if it's a publiclicly accessible bucket, it will always be HTTP, so it's no different from downloading the JAR from the other HTTP resources

Reply:
That's true, but users' first instinct would be to fill in the bucket fields if the source is a bucket.